commands/testdata/vcsclientmock.go

```diff
@@ -0,0 +1,338 @@
+// Code generated by MockGen. DO NOT EDIT.
```

Comment:
This file is autogenerated and therefore I suggest not adding it to the repository.